independent-publisher-connectors/ClickUp Team Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1231 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "ClickUp Team Manager",
+    "description": "Connects to the ClickUp team to manage tasks, folders, and lists from within Power Platform",
+	    "contact": {

```

Comment:
@marsurice Please remove extra spacings in all three artifacts.

Reply:
@marsurice  Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
@marsurice This has not addressed or fixed, Please revisit the comment and push the changes.